.ci/build.sh

```diff
@@ -55,6 +55,16 @@ function build() {
     # stop early for invalid maven version and branch/tag combination
     pmd_ci_maven_verify_version || exit 0
 
+    # allow snapshot dependencies in release candidate builds:
+    if [[ "${PMD_CI_MAVEN_PROJECT_VERSION}" == *-rc* ]]; then
+        PMD_MAVEN_EXTRA_OPTS+=(-Denforcer.skip=true)
+    fi
```

Comment:
I think, we shouldn't do this. Keeping snapshot dependencies makes the build not reproducible anymore. E.g. 7.0.0-rc1 is not reproducible with the following error:

```
[ERROR] Failed to execute goal on project pmd-cli: Could not resolve dependencies for project net.sourceforge.pmd:pmd-cli:jar:7.0.0-rc1: Could not find artifact net.sourceforge.pmd:pmd-ui:jar:7.0.0-SNAPSHOT in default (https://repo.maven.apache.org/maven2) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project pmd-cli: Could not resolve dependencies for project net.sourceforge.pmd:pmd-cli:jar:7.0.0-rc1: Could not find artifact net.sourceforge.pmd:pmd-ui:jar:7.0.0-SNAPSHOT in default (https://repo.maven.apache.org/maven2)
```

Since I released the pmd-designer yesterday, we can now use a released version and this workaround is not needed anymore.